### PR TITLE
Vp9 unit

### PR DESCRIPTION
**Pull request by wudping**
use base's setFormat and ensureProfile, and implement decode API unittest for VP9.

### CODE REVIEW

File: decoder/DecoderApi_unittest.cpp [hunk 1]
```diff
@@ -230,7 +241,7 @@ TEST_P(DecodeApiTest, Format_Change)
 
         YamiStatus status = decoder->decode(&buffer);
         if (prevFormat.width != info.m_width
-            || prevFormat.width != info.m_width) {
+            || prevFormat.height != info.m_height) {
```
Comment (xuguangxin) [inline decoder/DecoderApi_unittest.cpp:244]: Please submit another commit for this typo.

**Comment by wudping** [comment]
it's done, @xuguangxin, thanks for your suggestion.

**Review by xuguangxin — APPROVED** [review]
great! thanks.

**Comment by wudping** [comment]
add jpeg unittest relative patches, please help to review. Thanks.
